bindings/r/src/TRInterface.cxx

```diff
@@ -44,6 +49,15 @@ TRInterface::TRInterface(const Int_t argc, const Char_t *argv[], const Bool_t lo
    statusEventLoop = kFALSE;
    std::string osname = Eval("Sys.info()['sysname']");
    //only for linux/mac windows is not supported by ROOT yet.
+#if defined(HAS_X11)
+   if (!gROOT->IsBatch()) {
+      if (gEnv->GetValue("X11.XInitThread", 1)) {
```

Comment:
Why is "TGX11::OpenDisplay" not called (and thus that code) in this use case while it is called in all other cases?

Reply:
Hello Philippe,
this is to call R plots, not ROOT plots.
The system is producing
`
[[xcb] Unknown sequence number while processing reply
[xcb] Most likely this is a multi-threaded client and XInitThreads has not been called
[xcb] Aborting, sorry about that.
root.exe: ../../src/xcb_io.c:641: _XReply: Assertion `!xcb_xlib_threads_sequence_lost' failed.]
`
If you are calling R plots after call ROOT plots there is no problems, 
but If you are calling R plots without call any ROOT plot this is required.